.eslintrc.js

```diff
@@ -54,6 +53,7 @@ module.exports = {
                 "ignoreProperties": true
             }
         ],
+        "@typescript-eslint/no-unused-vars": ["error", { "varsIgnorePattern": "^_", "argsIgnorePattern": "^_" }],
```

Comment:
It occurs to me that we use `_` as the string translate function (due to xgettext history), so I'm thinking we should avoid having a pattern that might unintentionally encourage the use of `_` exactly as a "ignore this" variable.  Specifically I'm thinking we should have our pattern make sure at least one character follows the leading `_` so we don't inadvertently shadow `_`. If I understand the pattern usage correctly, I think if we change the patterns to something like `^_[a-zA-Z_]` would do the trick?

Reply:
That makes sense. For the arguments modified in this commit, would you prefer we just drop them rather than using the underscore names: https://github.com/online-go/online-go.com/pull/1641/commits/e91f6ff2032a3319012d280e1a6210cbe9da40c1?

Reply:
Yeah lets just get rid of those, not sure why I felt the need to have parameters in there that I just ignored.

Reply:
Updated. Here's a weird thing I just noticed: the underscore at this line is actually unused but didn't get flagged as such even though it shouldn't match the pattern: https://github.com/dsaxton/online-go.com/blob/e1f61597eb93124d0f85f0b70e9be5db3317ebfe/src/lib/translate.ts#L376. But apparently that's correct according to JavaScript regex, or am I missing something here?

```javascript
> /_/.test("^_[a-zA-Z_]")
true
```

I think this should be false according to the Python `re` library at least.

Reply:
Updated, I wonder why this undescore doesn't get flagged as unused though: https://github.com/dsaxton/online-go.com/blob/e1f61597eb93124d0f85f0b70e9be5db3317ebfe/src/lib/translate.ts#L376 :thinking:

Reply:
By default, yes (but it's configurable)

>### args
>
>The args option has three settings:
>
>    after-used - unused positional arguments that occur before the last used argument will not be checked, but all named arguments and all positional arguments after the last used argument will be checked.
>    all - all named arguments must be used.
>    none - do not check arguments.

https://eslint.org/docs/rules/no-unused-vars#args